test/e2e/flowaggregator_test.go

```diff
@@ -282,8 +285,8 @@ func checkIntraNodeFlows(t *testing.T, data *TestData, podAIPs, podBIPs *PodIPs,
 	}
 }
 
-func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs, podEIPs *PodIPs, isIPv6 bool) {
-	svcB, svcC, err := createPerftestServices(data, isIPv6)
+func testHelper(t *testing.T, data *TestData, isIPv6 bool) {
+	svcB, svcC, svcD, svcE, err := createPerftestServices(data, isIPv6)
```

Comment:
should we also move `createPerftestServices ` to `TestFlowAggregator`, and maybe simplify the code by creating one Service for each Pod indiscriminately?

Reply:
I think it should be kept in testHelper as the services need to be created based on the isIPv6 condition?

Reply:
it doesn't have to be done in this PR necessarily, but it would be better to create the Services once, in `TestFlowAggregator`. The `IPFamilies` field in the `ServiceSpec` is a list, and based on what the cluster supports, it can be set to `[IPv4]`, `[IPv4, IPv6]`, or `[IPv6]`. Then the test can access the service using the correct IP. What do you think?

Reply:
It should be a good idea as `IPFamilies` is a list. Since `CreateServiceWithAnnotations` is used by many functions, I think it might be better to open another PR to make this change.

Reply:
I opened another PR to change the type of `IPFamily`. https://github.com/antrea-io/antrea/pull/5690